macOS/DuckDuckGo/Tab/TabExtensions/HistoryTabExtension.swift

```diff
@@ -115,6 +131,17 @@ final class HistoryTabExtension: NSObject {
             }
         }.store(in: &cancellables)
 
+        scriptsPublisher.sink { [weak self] scripts in
+            Task { @MainActor in
+                self?.historyViewUserScript = scripts.historyViewUserScript
+                self?.historyViewUserScript?.webView = self?.webView
+            }
+        }.store(in: &cancellables)
+
+        webViewPublisher.sink { [weak self] webView in
+            self?.webView = webView
+        }.store(in: &cancellables)
+
```

Comment:
For `HistoryView.StyleClient` 's `pushMessage(name:)` API to work, we need a reference to the WebView itself. 

This mechanism mirrors `NewTabPageTabExtension` to the letter